runtime/makelib/targets.mk.aix.inc.ftl

```diff
@@ -158,7 +158,7 @@ $(patsubst %.s,%.o,$(filter %.s,$(UMA_FILES_TO_PREPROCESS))) : %$(UMA_DOT_O) : %
 
 ifdef UMA_TREAT_WARNINGS_AS_ERRORS
   ifndef UMA_SUPPRESS_WARNINGS_AS_ERRORS
-    CFLAGS += -qhalt=w
-    CXXFLAGS += -qhalt=w
+    CFLAGS += -qhalt=e
```

Comment:
What does this fix? Although you said `likely not feasible as long term solutions`, we don't want this change even as a short term solution as it's hard to get back to a state with no warnings once warning are allowed to creep in.

Reply:
A better way to do this, although still better to fix the warnings, is to disable UMA_TREAT_WARNINGS_AS_ERRORS for only the subdirectory that needs it.

Reply:
This was intended to fix this error: 
```16:13:46  jnitest.c:1020:1: error: control reaches end of non-void function [-Werror,-Wreturn-type]```
At that point my priority was getting the build to succeed so that we could compare performance between xlC and xlclang, but since that's no longer the case, should I just scrap this change entirely?

Reply:
Yes, please remove this part of the change. You can do this for your personal builds, but we should fix the jnitest.c warning properly. Which can be done in another PR.